bundles/org.openhab.binding.velbus/README.md

```diff
@@ -1,140 +1,267 @@
 # Velbus Binding
 
-The Velbus binding integrates with a [Velbus](https://www.velbus.eu/) system through a Velbus configuration module (VMBRSUSB, VMB1USB or VMB1RS) or a network connection (TCP/IP).
+The Velbus binding integrates with a [Velbus](https://www.velbus.eu/) system through a Velbus
+configuration module (VMBRSUSB, VMB1USB or VMB1RS) or a network connection (TCP/IP).
```

Comment:
Please revert the line break. One sentence per line makes it easier to view diffs.

Please check others as well.